### ISSUE TITLE
Set routes in url

### ISSUE THREAD

**Issue by romangg** [anydistro/bxt#109]
At the moment the app is always displaying the same main url.

It should be possible to have some routing logic though as we have a tree-like structure:

* Main menu items: `bxt.manjaro.org/{packages,compare,log,admin}`
* In packages (example): `bxt.manjaro.org/packages/unstable/core/x86_64`

Of course not only display should work but also navigating to such a view directly by entering the url.

**Comment by LordTermor** [comment]
It actually shows 

> Main menu items: bxt.manjaro.org/{packages,compare,log,admin}

for me both on Firefox and Vivaldi. Packages will be more tricky to implement as it uses custom navigation logic.

**Comment by romangg** [comment]
True, main menu items show. But packages paths would be nice to have additionally.
